tinygrad/state.py

```diff
@@ -43,16 +43,16 @@ def get_state_dict(obj, prefix:str='', tensor_type=Tensor) -> Dict[str, Tensor]:
   return state_dict
 def get_parameters(obj) -> List[Tensor]: return list(get_state_dict(obj).values())
 
+@Timing("loaded weights in ", lambda et_ns: f", {GlobalCounters.mem_used/1e9:.2f} GB loaded at {GlobalCounters.mem_used/et_ns:.2f} GB/s")
```

Comment:
Why did this change? Is this needed as part of this PR?

Reply:
I added `contextlib.ContextDecorator` to Context which makes it possible to use it as a decorator.
I was not sure if that would be a wanted feature but its only a `+1` line change so thought to suggest it.

And as Timing was another helper contextmanager it was a `+0` line change to add it to it as well.

I checked how `Timing` was used and noticed this one case which would be a natural fit for the decorator Timing pattern so felt natural to show how this feature would affect the code if used.